app/src/main/res/values/dimens.xml

```diff
@@ -19,6 +19,9 @@
     <dimen name="title_font_size">31sp</dimen>
     <dimen name="subtitle_font_size">16sp</dimen>
     <dimen name="big_font_size">18sp</dimen>
+    <dimen name="exposure_summary_text">20sp</dimen>
+    <dimen name="padding_content">16dp</dimen>
+    <dimen name="small_padding_content">8dp</dimen>
```

Comment:
nit: use padding_normal, padding_small

Reply:
Yes, we should refactor later to use only one source of the same padding size. Or in general, the same sizes in resources in order to avoid duplication.